src/FTPClient.jl

```diff
@@ -16,6 +16,7 @@ type FTPClientError <: Exception
        lib_curl_error::UInt32
 end
 
+@enum FTP_MODE ascii_mode binary_mode
```

Comment:
Since we have a function called `ascii` and `binary` there will be some issues. As shown here,
```
julia> function foo(a)
       a
       end
foo (generic function with 1 method)

julia> foo(2)
2

julia> @enum FOOBAR foo bar
ERROR: invalid redefinition of constant foo
 in eval(::Module, ::Any) at ./boot.jl:267
```

I could get rid of the functions since I don't think there's any reason to use them.